cmds/dmesg/dmesg_test.go

```diff
@@ -5,12 +5,17 @@
 package main
 
 import (
+	"os"
 	"testing"
 
 	"github.com/u-root/u-root/pkg/testutil"
 )
 
 func TestDmesg(t *testing.T) {
+	if os.Getuid() != 0 {
```

Comment:
Split this into a different PR or commit. We can try to get it in faster because it's also affecting #1162.